src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -631,7 +632,7 @@ static inline int MPIDI_NM_mpi_win_lock(int lock_type, int rank, int assert, MPI
 #define FUNCNAME MPIDI_NM_mpi_win_unlock
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t *addr)
+static inline int MPIDI_NM_mpi_win_unlock(int rank, MPIR_Win * win, MPIDI_av_entry_t * addr)
```

Comment:
Well... yeah... at least it isn't a bad idea to make the pointer style consistent in this line. I just noticed that `MPIR_Win * win` had a space before `win` but `addr` was not.